package.json

```diff
@@ -206,6 +210,11 @@
 					"command": "launchdarkly.refreshEntry",
 					"when": "view == launchdarklyFeatureFlags",
 					"group": "navigation"
+				},
+				{
+					"command": "extension.configureLaunchDarkly",
+					"when": "!!config.accessToken && !!config.project && !!config.env",
```

Comment:
should the first `&&` here be a `|`? If there's no access token, it doesn't really matter if project & env is configured. Not sure how that could happen though

Reply:
The behavior I was going for was that the icon should only show up when you're already configured. It doesn't show up when you're not configured because the welcome content with the "configure" button will already be displayed.